android-components/components/service/contile/src/main/java/mozilla/components/service/contile/ContileTopSitesProvider.kt

```diff
@@ -49,26 +57,24 @@ class ContileTopSitesProvider(
     // The last modified time of the disk cache.
     @VisibleForTesting
     @Volatile
-    internal var diskCacheLastModified: Long? = null
+    internal var diskCacheMaxAge: Long? = null
```

Comment:
❓ I find I'm getting confused by `maxCacheAgeInMInutes` and `diskCacheMaxAge`.  Is it possible to have some helper functions for `maxCacheAgeInMinutes` so we don't need a separate variable `diskCacheMaxAge`?

Reply:
Added the `CacheState` for extra clarity, thank you for the suggestion!